velox/functions/prestosql/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ add_library(
   MapZipWith.cpp
```

Comment:
Please add document for this function? See velox/docs/functions.rst for example

Reply:
+1